### PR TITLE
[data-plane report] 3th update

### PR DESCRIPTION
**Pull request by JackTn**
1. add operationId correct link
2. add PathInPayload correct link
3. add schema path correct link

### CODE REVIEW

File: lib/templates/baseLayout.mustache [hunk 1]
```diff
@@ -472,9 +472,7 @@
                         {{#getGeneralErrorsGen}}
                             <tr>
                                 <td>{{operationInfo.operationId}}
-                                {{#coverageResultsForRendering}}
```
Comment (raych1) [inline lib/templates/baseLayout.mustache:475]: Why is this tag removed?

File: lib/swaggerValidator/trafficValidator.ts [hunk 2]
```diff
@@ -158,12 +163,22 @@ export class TrafficValidator {
 
   public async validate(): Promise<TrafficValidationIssue[]> {
     let payloadFilePath;
+    const swaggerOpts: SwaggerLoaderOption = {
+      setFilePath: false,
+    };
+    this.swaggerLoader = inversifyGetInstance(SwaggerLoader, swaggerOpts);
+    const spec = await this.swaggerLoader.load(this.specPath);
```
Comment (raych1) [inline lib/swaggerValidator/trafficValidator.ts]: `specPath` could be a folder, use `swaggerFile` in `for` loop.

File: lib/templates/baseLayout.mustache (hunk 1 above)
Comment (JackTn) [reply lib/templates/baseLayout.mustache:475]: got  spec link data for render with operationId from getGeneralErrorsGen not from coverageResultsForRendering

File: lib/swaggerValidator/trafficValidator.ts [hunk 3]
```diff
@@ -306,44 +345,39 @@ export class TrafficValidator {
...
-        apiVersion: getApiVersionFromFilePath(key),
-        coveredOperaions: coveredOperaions,
-        coverageRate: coverageRate,
-        unCoveredOperations: value.length - coveredOperaions,
-        totalOperations: value.length,
-        validationFailOperations: validationFailOperations,
-        unCoveredOperationsList: sortedUnCoveredOperationsList,
-        unCoveredOperationsListGen: sortedUnCoveredOperationsListGen,
-      });
+      for (let i = 0; i < unCoveredOperationsListFormat.length - 1; i++) {
```
Comment (raych1) [inline lib/swaggerValidator/trafficValidator.ts:355]: Can you add comment for this sorting by ascending order?

Comment (JackTn) [reply lib/swaggerValidator/trafficValidator.ts:355]: added

File: lib/swaggerValidator/trafficValidator.ts (hunk 2 above)
Comment (JackTn) [reply lib/swaggerValidator/trafficValidator.ts]: deleted

**Review by raych1 — APPROVED** [review]

**Review by raych1 — APPROVED** [review]

**Pull request merged**